packages/newsroom-signup/src/NewsroomProfile/LearnMoreButton.tsx

```diff
@@ -78,45 +78,78 @@ export class LearnMoreButton extends React.Component<LearnMoreButtonProps, Learn
-              of their ability.
+              The Civil Registry is the online destination where prospective Civil Newsrooms apply to be part of the
+              network.
             </Paragraph>
-            <SectionHeader>What is the Civil Registry</SectionHeader>
             <Paragraph>
-              Civil Registry is the destination to which prospective Civil Newsrooms must apply to access publishing
-              rights on the Civil platform. You can apply to the Civil Registry after creating your Newsroom Smart
-              Contract.
+              Newsrooms on the Civil Registry have been vetted by the community and deemed it as having a credible,
+              journalistic practices as outlined in the{" "}
+              <a href="https://civil.co/constitution/" target="_blank">
+                Civil Constitution
+              </a>.
             </Paragraph>
+
+            <SectionHeader>Civil Tokens</SectionHeader>
             <Paragraph>
-              Being included in the Civil Registry means that the community has vetted a Newsroom and deemed it as
-              having a credible, journalistic mission, and that it has pledged to adhere to the journalistic ethics
-              outlined by the Civil Constitution.
+              Civil tokens (CVL) enable the "community-owned and operated" Civil model to work. A token is a piece of
+              software, it’s not a physical coin. But it does have value and represents a piece of ownership in the
+              network.
             </Paragraph>
+            <Paragraph>
+              For those that want to get more technical, CVL is a utility token based on the ERC20 protocol. It’s a
+              value stored in a decentralized database that’s managed by Civil’s smart contracts, which allows the Civil
+              platform to interact with the Civil token. CVL is the software that bridges Civil with the Ethereum
+              blockchain.
+            </Paragraph>
+
+            <SectionHeader>Ether (ETH)</SectionHeader>
+            <Paragraph>
+              Ether (ETH) is different cryptocurrency than CVL but you will need some to to pay for certain actions on
+              Civil like sending tokens. That takes a small amount of computing effort, called Gas, which must be paid
+              for in ETH. When sending a transaction, you set a gas limit. You must set this limit carefully, because if
+              you set it too low and the transaction fails as a result, you will lose the gas / ETH that you spent.
+            </Paragraph>
+
             <SectionHeader>Applying to the Civil Registry</SectionHeader>
             <Paragraph>
-              You will need to deposit 0,000 CVL token with your application. This is to state the seriousness of your
-              intent to the community.
+              You will need to deposit $1,000.00 worth of CVL tokens with your application. This is to signal the
```

Comment:
not necessarily true

Reply:
we at the very least shouldnt use decimal places

Reply:
what's not true, depositing 1,000 to state their intent? agreed on the decimals.

Reply:
its like close to 1000 dollars now but will drift over time.  We can't change the deposit without a community vote, so theres no guarantee that its 1000 dollars at any point in the future